src/backend/core/api/custom_collector.py

```diff
@@ -0,0 +1,201 @@
+"""Custom Business metrics collector for Docs."""
+
+from datetime import timedelta
+
+from django.conf import settings
+from django.db import DatabaseError
+from django.db.models import Count, Max, Min, Q
+from django.utils.timezone import now
+
+from prometheus_client.core import GaugeMetricFamily
+from prometheus_client.registry import Collector
+
+from core import models
+
+
+class CustomCollector(Collector):
+    """
+    Custom Prometheus metrics collector for various application metrics.
+
+    Metric naming follows Prometheus best practices:
+    - leading application (namespace) prefix configured via settings
+    - snake_case names with explicit unit/type suffix where relevant
+    - use labels inside the metric name
+    """
+
+    def _get_metric_name(self, name):
+        """
+        Get the Prometheus metric if PROMETHEUS_METRIC_NAMESPACE is set.
+        """
+        namespace = getattr(settings, "PROMETHEUS_METRIC_NAMESPACE", "")
```

Comment:
```suggestion
        namespace = settings.PROMETHEUS_METRIC_NAMESPACE
```

The setting exists in the settings module, you can directly use it.